### PR TITLE
URLConfig not in strax

### PR DESCRIPTION
**Pull request by ershockley**
There is a small bug here, I presume from this PR originating in strax or vice versa? 

In current release, I get:

```
>>> from straxen.url_config import URLConfig
[lots of UserWarnings...]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/cvmfs/xenon.opensciencegrid.org/releases/nT/development/anaconda/envs/XENONnT_development/lib/python3.8/site-packages/straxen/url_config.py", line 168, in <module>
    @strax.URLConfig.register('cmt')
AttributeError: module 'strax' has no attribute 'URLConfig'
```

This PR should fix this bug.

### CODE REVIEW

**Comment by ershockley** [comment]
I should add that this is in the current development container.

**Review by jmosbacher — APPROVED** [review]
Thanks Evan for catching this! we moved this code from strax to straxen rather hastily prior to merging and somehow this got unnoticed. I also added an import of the url_config in the package init file.

**Comment by JoranAngevaare** [comment]
Maybe this would also be a good time to add some tests 👼 ?

**Comment by jmosbacher** [comment]
> Maybe this would also be a good time to add some tests angel ?
Yes indeed!

**Comment by jmosbacher** [comment]
@JoranAngevaare I added some basic tests. for now no fancy hypothesis testing, just some hardcoded examples.

**Comment by jmosbacher** [comment]
No addresses #769

**Comment by jmosbacher** [comment]
now addresses #782

**Comment by jmosbacher** [comment]
OK tests are finally passing. The testing envs were running pymongo 4.0 which introduced some breaking changes. Hopefully I found all of the effects on our code.

File: requirements.txt
Comment (JoranAngevaare) [inline requirements.txt:3]: Is this still needed after the changes you made?

File: tests/test_url_config.py
Comment (JoranAngevaare) [inline tests/test_url_config.py:20]: Maybe also add a test for the `protocol_descr` and `fsspec`, mostly because I think these tests are actually super instructive examples of how the code should be used.

**Review by JoranAngevaare — APPROVED** [review]
Thanks Yossi, looks good and it's nice to have some tests and examples in place!

File: requirements.txt
Comment (jmosbacher) [reply requirements.txt:3]: only if you want to be extra careful. If I successfully found all the breaking changes then its no longer needed.

File: tests/test_url_config.py
Comment (jmosbacher) [reply tests/test_url_config.py:21]: added a test for fsspec protocol. not sure how to test protocol_descr method since its just a helper method to quickly see what protocols are registered and their call signature

File: requirements.txt
Comment (JoranAngevaare) [reply requirements.txt:3]: based on strax' tests I think it's indeed good to add this requirement

**Comment by JoranAngevaare** [comment]
@jmosbacher , can we merge?

**Comment by jmosbacher** [comment]
@JoranAngevaare yeah im not planning on adding anything. @ershockley did you want to add anything else before we merge this?

**Comment by ershockley** [comment]
Sorry I did not see this until now. It looks good -- thanks a lot!

**Pull request merged**
